cloudify_rest_client/tenants.py

```diff
@@ -80,7 +80,7 @@ def create(self, tenant_name):
 
         return Tenant(response)
 
-    def add_user(self, username, tenant_name, role=None):
+    def add_user(self, username, tenant_name, role):
         """Add user to a tenant.
 
         Optionally, set the role of the user in the context of the tenant in
```

Comment:
Please remove the "Optionally" word from the docstring now that `role` is required